Makefile

```diff
@@ -21,6 +21,20 @@ BUCKET ?= gs://k8s-staging-kwok-gcb
 
 GIT_TAG ?= $(shell git describe --tags --dirty --always)
 
+BASE_REF ?= $(shell git rev-parse --abbrev-ref HEAD)
```

Comment:
Isn't it a bit much that every commit on a branch builds the image once?
https://github.com/kubernetes/test-infra/pull/27128/files#diff-658f73e8fc0958a94f1af5d908b874ce2331bcad55cec424bd91172c1924775aR12